### DISCUSSION TITLE
botman is reapting handle function after one message

### DISCUSSION THREAD

**Discussion by Palak7y7y** [botman/botman#1385] [General]
this is my botman handle after  showing this message "Hey There! How are you?" when I reply like good or bad then it will repeat the handle function instead of giving me a  this reply "'You said:  good"

Please give me a solution if you also face the similar situation 

public function handle()
    {
        $botman = app('botman');
        $botman->hears('{message}', function($botman, $message) {
            // dd($message);
            if ($message == 'hello') {
                $this->askName($botman);
            }else{
                $botman->reply("Type 'hello' for demo ...");
            }
        });
        $botman->listen();
    }
    public function askName($botman)
    {
        $botman->ask('Hey There! How are you?', function(Answer $answer) {
            $ans = $answer->getText();
            $botman->say('Hello, ' . $ans . '! How can I assist you?', function() use ($botman) {
            // You can continue the conversation here
            $botman->ask('What can I do for you?', function(Answer $answer) {
            // Handle user's response here
            $response = $answer->getText();
            $botman->reply('You said: ' . $response);
            });
        });
    });

**Comment by filippotoso** [comment]
Are you using a Conversation object?

**Reply by Palak7y7y** [reply]
yeah  i try that but issue is still there. i use botman before in 4-5 projects but this is the first I am facing this kind of problem also it works on  localhost properly
